src/omero_cli_render.py

```diff
@@ -204,7 +203,7 @@ def __init__(self, image):
 
         self.range = image.getPixelRange()
         self.channels = map(lambda x: ChannelObject(x),
-                            image.getChannels(noRE=True))
+                            image.getChannels(noRE=False))
```

Comment:
It might be worth tracking down the original commit here, since this may have been for performance reasons. @manics : can you remember?

Reply:
No idea

Reply:
The problem is, with `noRE` the StatsInfo object is unloaded. Alternatively could just catch that, but then `render info` doesn't make sense, because it basically doesn't give you much information then.